Improve props-II in React (#1519)
enkidevs/curriculum: javascript/react/props-ii/specify-a-single-child.md

---
## Content

> Making use of the `propTypes`, you can enforce a *warning* that is fired when you pass more than one child to a component.

Reviewer: ```suggestion
Making use of the `propTypes`, you can show a *warning* or throw an *error*, when you pass more than one child to a component.
```

The validator used is `PropTypes.element`.

Suppose we have the following scenario:
```jsx
class NewComponent extends React.Component {
  render() {
    return <div>{this.props.children}</div>;
  }
}
```

> To make sure `this.props.children` is **exactly** a single element, we suffix the validation with `isRequired`:

Reviewer: ```suggestion
To throw an error if anything but exactly 1 element is passed as `children`, we suffix the `PropTypes` validation with `isRequired`:
```
